test/browser-child.test.js

```diff
@@ -1,32 +1,32 @@
 'use strict'
-const test = require('tape')
+const test = require('node:test')
```

Comment:
I don't know if this is what we want to do. I think `tape` is used for some browser reason. @mcollina do you know?

Reply:
@jsumners @mcollina, any updates on this one?

Reply:
Yes I would keep tape for this is possible. This is run via https://github.com/pinojs/pino/blob/a42487988dfa2cf4624e38a46673657bd9772a1f/package.json#L11. But I'm not sure this has been exercised in a long time.

Reply:
Perfect, I'll close this pr